### PR TITLE
Add results and 404 page

### PR DESCRIPTION
**Pull request by Maianki**
## Added result page functionality and 404 page
This PR includes
- [ ] 404 page
- [ ] Result page functionality

### Thanks for reviewing my PR

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me! You can merge the PR

**Review by anik31 — APPROVED** [review]
The result & 404 page looks good. Good job.

suggestion - you can add a button on 404 page to return to home page

File: src/pages/404/PageNotFound.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./page-not-found.css";
+import notFoundImg from "../../assets/images/404.svg";
+
+export function PageNotFound() {
```
Comment (sainath7878) [inline src/pages/404/PageNotFound.jsx:5]: Appreciation: Good that you've implemented not found page

**Review by sainath7878 — APPROVED** [review]
Small PR, nothing to suggest. Functionality is working as expected. Code is neat and understandable.

**Comment by Maianki** [comment]
Thanks @ApurvaSawant11 @anik31 @sainath7878  for PR review

**Pull request merged**